tests/antithesis/test-template/robustness/main_test.go

```diff
@@ -88,16 +88,23 @@ func main() {
 	ctx := context.Background()
 	baseTime := time.Now()
 	duration := time.Duration(robustnessrand.RandRange(5, 60) * int64(time.Second))
-	testRobustness(ctx, hosts, persistedRequestdirs, reportPath, baseTime, duration)
+	testRobustness(ctx, t, hosts, persistedRequestdirs, reportPath, baseTime, duration)
 }
 
-func testRobustness(ctx context.Context, hosts, persistedRequestdirs []string, reportPath string, baseTime time.Time, duration time.Duration) {
+func testRobustness(ctx context.Context, t *testing.T, hosts, persistedRequestdirs []string, reportPath string, baseTime time.Time, duration time.Duration) {
 	lg, err := zap.NewProduction()
 	if err != nil {
 		panic(err)
 	}
+
+	r := report.TestReport{Logger: lg}
+	panicked := true
+	defer func() {
+		r.Report(t, panicked)
```

Comment:
Instead to changing the binary to be a test, can we just remove `t` in r.Report and instead just return error?

Reply:
Not totally sure I got it right.  TestReport.Report needs a *testing.T to be called.  I have removed that call for now as we have not set it up correctly to persist with the (optional?) environment variable.  Let me know if this is off..

Reply:
pushed the new commit in.  making it to nil for now.